tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/CustomAttributes/AttributeUsage/E_AttributeTargetIsClass02.fs

```diff
@@ -0,0 +1,23 @@
+open System
+
+[<AttributeUsage(AttributeTargets.Class)>]
+type ClassTargetAttribute() =
+    inherit Attribute()
+
+[<AttributeUsage(AttributeTargets.Interface)>]
+type InterfaceTargetAttribute() =
+    inherit Attribute()
+
+[<AttributeUsage(AttributeTargets.Struct)>]
+type StructTargetAttribute() =
+    inherit Attribute()
+
+[<InterfaceTarget>]
+[<StructTarget>]
+[<ClassTarget>]
+type Record = { Prop: string }
+
+[<ClassTarget>]
+[<InterfaceTarget>]
+[<Struct>]
```

Comment:
You can probably add the (legit) `StructTarget` attribute here for the symmetry with the record case.